descriptors/ModuleDescriptor-template.json

```diff
@@ -28,7 +28,7 @@
     },
     {
       "id": "inventory-view",
-      "version": "1.1"
+      "version": "13.0"
```

Comment:
Here the version should be "version": "2.0"

Reply:
And the inventory version should be "version": "13.0"

Reply:
changed